app.py

```diff
@@ -0,0 +1,6 @@
+import os
+from flaskr import create_app
+
+if __name__ == "__main__":
+    app = create_app()
+    app.run(host=os.environ["FLASK_HOST"], port=os.environ["FLASK_PORT"])
```

Comment:
I think you should just be able to set the host here to `0.0.0.0` and remove that environment variable from the Dockerfile